socorro/external/es/super_search_fields.py

```diff
@@ -2,16 +2,20 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+import json
+import os
 import datetime
 import elasticsearch
 
-from socorro.lib import (
-    BadArgumentError,
-    MissingArgumentError,
-    ResourceNotFound,
-)
+from socorro.lib import BadArgumentError, datetimeutil
 from socorro.external.es.base import ElasticsearchBase
-from socorro.lib import datetimeutil, external_common
+
+
+SUPER_SEARCH_FIELDS_JSON_PATH = os.path.join(
+    os.path.dirname(__file__),
+    'data',
+    'super_search_fields.json'
+)
```

Comment:
Pretty sure this isn't going to work in -stage/-prod since it gets installed as a package and I don't see anything that adds the `super_search_fields.json` file to the `data_files` section of `setup.py`.

I think this should work like you did siglists:

https://github.com/mozilla-services/socorro/blob/master/socorro/siglists/__init__.py#L41